README.md

```diff
@@ -180,15 +179,13 @@ building blocks of the Enclave*
 
 ### III. The Wallet Extension
 
-The missing link to achieving fully private transactions, while allowing end users to continue using their favourite wallets
-(like MetaMask) is a very thin component that has to be installed on the user machine.
-This component is responsible for the encryption/decryption of the traffic originating from an Obscuro node. It does that 
-by generating Viewing Keys behind the scenes.
+The missing link to achieving fully private transactions while allowing end-users to continue using their favourite 
+wallets (like MetaMask). This is a very thin component that is responsible for the encrypting and decrypting traffic 
```

Comment:
"for the encryption and decryption .." or "for encrypting or decrypting"

Reply:
Fixed.